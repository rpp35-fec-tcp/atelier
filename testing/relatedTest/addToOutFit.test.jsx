import React from 'react';
import AddToOutFit from '../../client/src/components/relatedItems&comparison/addToOutfit.jsx';
import ReactDOM from 'react-dom/client';
import fakeProductInfos from './fakeData.js';
import {act} from 'react-dom/test-utils';
import {render} from "@testing-library/react";

describe('Card', function() {
  test('render Add to out fit', () => {
    render(<AddToOutFit />);
  })
})