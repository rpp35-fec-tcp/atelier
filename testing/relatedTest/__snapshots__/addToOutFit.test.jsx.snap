// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddToOutFit Component snapshot test 1`] = `[Function]`;
